src/commands/run.js

```diff
@@ -194,6 +218,21 @@ export const handler = async (argv) => {
         status.stopAndPersist({ text: 'continuing ...' })
     }
 
+    /**
+     * stop tunnel if one was created in the beginning
+     */
+    if (tunnelProcess) {
+        status.start('Stopping Sauce Connect tunnel...')
+        await new Promise((resolve) => {
+            const shutdownTimeout = setTimeout(resolve, TUNNEL_SHUTDOWN_TIMEOUT)
```

Comment:
Shouldn't it reject in such a case?

Reply:
It is highly unlikely that this fails or times-out. If so we can just ignore the fact and should not let the test fail.

Reply:
Yeah, it should not fail the test but maybe logging that would be useful. Then one can clean up the tunnel process by themselves.

Reply:
good call